src/launch/lib/automation/processes/functions.py

```diff
@@ -68,3 +66,59 @@ def make_docker_aws_ecr_login(
             subprocess.run(["make", "docker/aws_ecr_login"], check=True)
     except subprocess.CalledProcessError as e:
         raise RuntimeError(f"An error occurred: {str(e)}") from e
+
+
+def git_config(
+    dry_run: bool = True,
+) -> None:
+    click.secho(f"Running make git config")
+    try:
+        if dry_run:
+            click.secho(
+                f"[DRYRUN] Would have ran subprocess: git config",
+                fg="yellow",
+            )
+        else:
+            subprocess.run(
+                ["git", "config", "--global", "user.name", "nobody"], check=True
+            )
+            subprocess.run(
+                ["git", "config", "--global", "user.email", "nobody@nttdata.com"],
+                check=True,
+            )
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(f"An error occurred: {str(e)}") from e
+
+
+def start_docker(
+    dry_run: bool = True,
+) -> None:
+    click.secho(f"Starting docker if not running")
+    try:
+        if not is_docker_running():
+            if dry_run:
+                click.secho(
+                    f"[DRYRUN] Would have started docker daemon",
+                    fg="yellow",
+                )
+            else:
+                subprocess.Popen(
+                    ["dockerd"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+                )
```

Comment:
This leaks file descriptors to the dockerd process. Maybe not a huge deal if a runner is thrown away and the process halted each time a pipeline is run, but definitely one of the gotchas of using the `Popen` API. Suggest using `close_fds=True` to keep this from happening.